### PR TITLE
 U4X-440 : Add end patient visit button in move to next service point action

### PR DESCRIPTION
**Pull request by jabahum**
## Requirements
- [ ] This PR has a title that briefly describes the work done including a [conventional commit](https://o3-docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) type prefix and a Jira ticket number if applicable. See existing PR titles for inspiration.

#### For changes to apps
- [ ]  My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide) and [**design documentation**](https://zeroheight.com/23a080e38/p/880723-introduction).

#### If applicable
- [ ] My work includes tests or is validated by existing tests.

## Summary

This PR adds the End Visit Button on the `Move to next service point ` action button

## Screenshots

Screenshot 2024-03-14 at 09 56 54

## Related Issue

https://metsprogramme.atlassian.net/browse/U4X-440

### CODE REVIEW

File: packages/esm-patient-queues-app/src/active-visits/change-status-move-to-next-dialog.component.tsx [hunk 1]
```diff
@@ -140,6 +140,36 @@ const ChangeStatusMoveToNext: React.FC<ChangeStatusDialogProps> = ({ patientUuid
...
 
+  // endVisit
+  const endVisitStatus = useCallback(
+    (event) => {
+      event.preventDefault();
+      const comment = event?.target['nextNotes']?.value ?? 'Not Set';
+      const status = 'Completed';
+      updateQueueEntry(status, provider, mappedQueueEntry?.uuid, contentSwitcherIndex, priorityComment, comment).then(
+        () => {
+          showToast({
```
Comment (akileng56) [inline packages/esm-patient-queues-app/src/active-visits/change-status-move-to-next-dialog.component.tsx:151]: After ending the Queue, Call the function to end the visit as well

**Comment by slubwama** [comment]
@jabahum indicate ticket number in the description and also in the heading of your pr

**Review by akileng56 — APPROVED** [review]

**Review by akileng56 — APPROVED** [review]

**Review by akileng56 — APPROVED** [review]

**Review by akileng56 — APPROVED** [review]

**Review by akileng56 — APPROVED** [review]
All good

**Review by akileng56 — APPROVED** [review]

**Review by slubwama — APPROVED** [review]

**Review by Mwanje — APPROVED** [review]

**Review by Mwanje — APPROVED** [review]
@jabahum thanks

**Pull request merged**